RequestService/RequestService.Repo/Migrations/20210223114301_ChangeLocationIDToNegative.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace RequestService.Repo.Migrations
+{
+    public partial class ChangeLocationIDToNegative : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DeleteData(
+                schema: "Lookup",
+                table: "SupportActivity",
+                keyColumn: "ID",
+                keyValue: 17);
+
+            migrationBuilder.DeleteData(
+                schema: "Lookup",
+                table: "SupportActivity",
+                keyColumn: "ID",
+                keyValue: 18);
+
+            migrationBuilder.DeleteData(
+                schema: "Lookup",
+                table: "SupportActivity",
+                keyColumn: "ID",
+                keyValue: 19);
+
+            migrationBuilder.DeleteData(
+                schema: "Lookup",
+                table: "SupportActivity",
+                keyColumn: "ID",
+                keyValue: 20);
+
+            migrationBuilder.Sql(@"
```

Comment:
Would be worth adding a Down migration for this as well.  Can be identical, i.e. multiply by -1